### DISCUSSION TITLE
Show Graphql compatibility in the Plugin Store

### DISCUSSION THREAD

**Discussion by sgtpenguin** [craftcms/console#66] [Plugin Store] [answered]
I use Craft headless with React+Next almost exclusively these days.  Sometimes plugins don't even mention that they support graphql in their store descriptions, leading me to click through to docs and command + f (my life is really hard I know).   Could be nice to have some kind of badge that indicates a plugin supports graphql if applicable visible from the search listing and detail page.

**Comment by brandonkelly** [comment]
This was previously requested by craftcms/.github#76, so marking as a duplicate. It’s a great idea though!

**Comment by brandonkelly** [comment, marked as answer]
Plugins can now specify their GraphQL compatibility, as well as whether they’ve been tested on Craft Cloud:

<img src="https://github.com/craftcms/console/assets/47792/61c691c3-01ef-435d-a355-e0b18264318b" width="485" height="191" alt="The GraphQL and Craft Cloud compatibility settings within Craft Console">

When enabled, they will be listed under “Compatibility” in the plugin store listing. See https://plugins.craftcms.com/ckeditor for an example.

**Reply by sgtpenguin** [reply]
nice!
